src/cascadia/Remoting/Peasant.idl

```diff
@@ -30,6 +30,11 @@ namespace Microsoft.Terminal.Remoting
         Windows.Foundation.DateTime ActivatedTime { get; };
     };
 
+	[default_interface] runtimeclass GetWindowLayoutArgs {

```

Comment:
Some woops-tabs here.

Reply:
This is what I get for using tabs all the time and relying on editorconfig to change things for me.